testfixtures/logcapture.py

```diff
@@ -145,17 +223,13 @@ def actual(self):
         This can be useful for making more complex assertions about logged
         records. The actual records logged can also be inspected by using the
         :attr:`records` attribute.
+
+        :rtype: List
+
         """
         actual = []
         for r in self.records:
-            if callable(self.attributes):
-                actual.append(self.attributes(r))
-            else:
-                result = tuple(self._actual_row(r))
-                if len(result) == 1:
-                    actual.append(result[0])
-                else:
-                    actual.append(result)
+            actual.append(self._rec_2_obj(r))
```

Comment:
```suggestion
            actual.append(self._actual_row(r))
```
...once `_rec_2_obj` has been collapsed into `_actual_row`